docusaurus.config.js

```diff
@@ -248,7 +248,7 @@ module.exports = {
       // Optional: Algolia search parameters
       //searchParameters: {},
       // Optional: path for search page that enabled by default (`false` to disable it)
-      searchPagePath: false,
+      //searchPagePath: false,
```

Comment:
Hi @vfalconisumo ! So this line is present in the [algolia configuration](https://github.com/SumoLogic/sumologic-documentation/pull/1248/files#). Can you make sure to work off of that branch when testing?